src/index.html

```diff
@@ -17,16 +17,48 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-block">
+      <div class="search-block__form">
+        <form
+          action="#"
+          data-qa="big"
+          class="search search--big"
+        >
+          <label
+            for="search-main"
+            class="search__label search__label--big"
+          >
+            <input
+              class="search__input search__input--big"
+              id="search-main"
+              type="text"
+              data-qa="keypress"
+              placeholder="Try “Los Angeles“"
+            />
+          </label>
+        </form>
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div class="search-block__form">
+        <form
+          action="#"
+          data-qa="small"
+          class="search search--small"
+        >
+          <label
+            for="search-secondary"
+            class="search__label search__label--small"
```

Comment:
There is no need to create big and small modifiers. Just let one of them be the default one, so you need only one modifier

Reply:
Thank you! I corrected my mistakes)